common/changes/office-ui-fabric-react/add-currentRenderedQuerystring_2018-09-24-23-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add currentRenderedQueryString to avoid discrepancy",
+      "type": "patch"
```

Comment:
Since we're changing the `IBaseExtendedPickerState` interface which is exported I think this should be `minor` not `patch`